src/locales/index.ts

```diff
@@ -1,7 +1,7 @@
 import { en } from './en'
```

Comment:
The `de` translation should be imported
```suggestion
import { en } from './en'
import { de } from './de'
```